lib/srv/discovery/discovery.go

```diff
@@ -960,6 +965,24 @@ func (s *Server) handleEC2RemoteInstallation(instances *server.EC2Instances) err
 			discoveryConfig: instances.DiscoveryConfig,
 			integration:     instances.Integration,
 		}, len(req.Instances))
+
+		for _, instance := range req.Instances {
+			s.awsEC2Tasks.addFailedEnrollment(
+				awsEC2FailedEnrollmentTaskKey{
+					accountID:   instances.AccountID,
+					integration: instances.Integration,
+					issueType:   usertasks.AutoDiscoverEC2IssueInvocationFailure,
```

Comment:
> usertasks.AutoDiscoverEC2IssueInvocationFailure

Nit: Should the issue type include something about SSM? Cause this is SSM installation failure, right? Maybe call it `AutoDiscoverEC2SSMFailure` or something.

Reply:
Renamed them to `AutoDiscoverEC2IssueSSMInvocationFailure`
I've also renamed other fields to match this pattern.